gap/prop.gi

```diff
@@ -8,6 +8,17 @@
 #############################################################################
 ##
 
+InstallMethod(IsChainDigraph, "for a digraph", [IsDigraph],
+function(digraph)
+  if DigraphNrEdges(digraph) = 0 and DigraphNrVertices(digraph) = 1 then
+    return true;
+  else
+    return IsConnectedDigraph(digraph) and InDegreeSet(digraph) = [0, 1]
+           and OutDegreeSet(digraph) = [0, 1]
+           and IsCycleDigraph(digraph) = false;
```

Comment:
This could be `not IsCycleDigraph(digraph)` instead of `IsCycleDigraph(digraph) = false`.